test/command_callback/bib_paths/dummy.bib

```diff
@@ -0,0 +1,11 @@
+@article{alevim2018,
+   author = {Juan Carlos P., Morgana K.},
+    title = "{Monte Carlo Methods for vim linter manipulation}",
+  journal = {ArXiv e-prints},
+archivePrefix = "arXiv",
+   eprint = {1803.01},
+ primaryClass = "math.PR",
+ keywords = {Vim - ALE Journal, 65C5, 930},
+     year = 2018,
+    month = jul
+}
```

Comment:
Make this file empty. The test won't read it.

Reply:
Done.